ports/espressif/boards/seeed_xiao_esp32c6/mpconfigboard.h

```diff
@@ -0,0 +1,22 @@
+// Micropython setup
+
+#define MICROPY_HW_BOARD_NAME       "Seeed Xiao ESP32-C6 4MB Flash 512KB SRAM"
+#define MICROPY_HW_MCU_NAME         "ESP32-C6FH4"
+
+#define MICROPY_HW_LED_STATUS (&pin_GPIO15)
+
+#define CIRCUITPY_BOARD_I2C         (1)
+#define CIRCUITPY_BOARD_I2C_PIN     {{.scl = &pin_GPIO23, .sda = &pin_GPIO22}}
+
+#define CIRCUITPY_BOARD_SPI         (1)
+#define CIRCUITPY_BOARD_SPI_PIN     {{.clock = &pin_GPIO19, .mosi = &pin_GPIO18, .miso = &pin_GPIO20}}
+
+// TXD0 and RXD0
+#define CIRCUITPY_BOARD_UART        (1)
+#define CIRCUITPY_BOARD_UART_PIN    {{.tx = &pin_GPIO16, .rx = &pin_GPIO17}}
+
+// For entering safe mode, use BOOT button
+#define CIRCUITPY_BOOT_BUTTON       (&pin_GPIO9)
+
+// Explanation of how a user got into safe mode
+#define BOARD_USER_SAFE_MODE_ACTION MP_ERROR_TEXT("Keep the BOOT button pressed and then connect to the computer via the data cable. Release the BOOT button after connecting to the computer.")
```

Comment:
I don't think this enters safe mode. I think it'll enter the bootrom. Please remove this. The default message should work.